### ISSUE TITLE
Dragonrider Boots, Ramstein's Lightning Bolts

### ISSUE THREAD

**Issue by fgsfdss** [LightsHope/issues#643]
- [Dragonrider Boots](https://web.archive.org/web/20050420164336/http://wow.allakhazam.com:80/db/item.html?witem=18102)

These are absent on Allakhazam before 2005, and considering the stats and the item ID being only 2 away from Band of Rumination, I think it's safe to say they should not be implemented yet.
[Patch 1.3](http://wowwiki.wikia.com/wiki/Patch_1.3.0):
>Rend's treasure table has been modified. 

- [Ramstein's Lightning Bolts](https://web.archive.org/web/20050222123001/http://wow.allakhazam.com:80/db/item.html?witem=13515)

Should deal 500 damage in a wider radius (unsure which).
[Patch 1.3](http://wowwiki.wikia.com/wiki/Patch_1.3.0):
>Ramstein's Lightning Bolts effect reduced to 200 points of damage and a 10-yard radius.

**Comment by Sabre142** [comment]
1. Probably correct, needs more research.

2. That is from the WoW beta before release.

3. Same as 2.

**Comment by fgsfdss** [comment]
You're right. Patch 1.3 was in March 2005 and the Allakhazam edits are from February.
